plan.go

```diff
@@ -34,7 +34,24 @@ func pairObjectsByCriteria(srcObjects []runtime.Object, dstObjects []runtime.Obj
 		pairs = append(pairs, pair)
 	}
 
-	//todo: iterate over dstObjects
+	for i := range dstObjects {
```

Comment:
lame that you're stuck using indexes everywhere. Is it idiomatic go?

Reply:
yeah this is apparently a common Go issue - there's another way to reference the object, but it's slightly more verbose